contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,115 @@
+
+When we launch a new product, or a project on an existing team becomes too large for the current team, we often spin out a new small team. This allows us to push forward with new ideas and keep shipping without sacrificing quality.
+
+Right now, for example, we're in the process of scaling support by moving our support engineers out of the Customer Success team and into a new [Customer Comms Team](/teams/customer-comms). We're [recruiting support engineers](/careers/support-engineer for that team, btw.
+
+We're also creating a new team with the mission of making our customer data platform (CDP) a first-class product – a project spun out from a Hackathon project at our recent all-company [offsite in Mykonos](/blog/mykonos-hackathon).
+
+
+## The golden rules of small teams
+
+The small teams structure is core to how we've [designed our company for speed](https://newsletter.posthog.com/p/how-to-design-your-company-for-speed), but they only work if you follow these rules. 
+
+### 1. They need to be genuinely small
+
+The first rule of small teams? They’ve got to be small – two to six people. More than this and you have a department, which is what we're trying to avoid. Less than two people and, well, you don’t have a team. 
+
+Small teams are effectively one pizza teams, as opposed to the [two-pizza teams](https://docs.aws.amazon.com/whitepapers/latest/introduction-devops-aws/two-pizza-teams.html) idea popularized by Amazon. 
+
+The overall goal of small teams is to own an area of the product or company and be structured like an early-stage startup. Sure, these small teams don't have all the functions of a startup, but their spirit and approach to work is exactly the same. 
+
+A lot of the people on our small teams are product engineers – i.e. engineers who also talk to users, and own product decisions. And small teams aren't just for product – our marketing team is a small team, as are customer success, and people and operations. 
+
+This startup-made-of-startups structure minimizes the number of centralized processes and the need for lots of layers of management. It biases to [the maker’s schedule](https://www.paulgraham.com/makersschedule.html) – and makers get shit done.
+
+### 2. They run themselves
+
+Each small team runs its own retrospective and sprint each week, with notes taken and shared in GitHub for the entire company to see.
+
+Small teams also make the final call on which features get into production, with no need for external quality assurance or control. And they can [merge whenever](https://posthog.com/handbook/engineering/development-process#merging).
+
+Each quarter, every small team at PostHog creates objectives and key results, or OKRs. Our executive team will give feedback on this to keep everyone on track. Beyond that, during each quarter each small team is tasked with maintaining a prioritized roadmap to help them achieve their objectives. Each small team is also tasked with speaking to customers. 
```

Comment:
```suggestion
Each quarter, every small team at PostHog outline their goals and projects for that quarter. We prefer goals orientated around what teams will ship, rather than more abstract goals like "increase conversions by 10%". Our executive team will give feedback on this to keep everyone on track. 
```